backend/api/Startup.cs

```diff
@@ -80,7 +80,7 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddMapster(options =>
             {
-                TypeAdapterConfig.GlobalSettings.Default.IgnoreNonMapped(true);
+                options.Default.IgnoreNonMapped(true);
```

Comment:
By default every property needs to be mapped.  This will allow us to control model versions.  Without it if we added a property to a model it would automatically include it in the output.